src/body-bottom/tracking-internal.js

```diff
@@ -112,7 +112,7 @@
 		expireDate = new Date(expireDate.getTime() + 1000 * 60 * 30);
 		document.cookie = `tracking_session_id=${window.sessionId}; expires=${expireDate.toGMTString()};` +
 			`domain=${cookieDomain}; path=/;`;
-		document.cookie = `pv_number=${window.pvNumber}; expires=${expireDate.toGMTString()}; path=${scriptPath};`;
+		document.cookie = `pv_number=${window.pvNumber}; expires=${expireDate.toGMTString()}; path=${scriptPath || '/'};`;
```

Comment:
in what cases the scriptPath is not set? if you set it to `/` then this pv counter will increase for all lang variants of the wiki

Reply:
`scriptPath` is empty for all English communities